test/TEST-57-RELOADING-RESTART/Makefile

```diff
@@ -0,0 +1 @@
+../TEST-01-BASIC/Makefile
```

Comment:
hmm, TEST-57 is already taken by #19322

Reply:
I've renamed to TEST-59, seems TEST-58 has been taken in the interim.